webrecorder/browsertrix
# Migrate crawl config frontend -> workflow

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/653:
- Renames "Crawl Configs" to "Workflows" in both user-facing text and refactored code
- Renames `/crawl-configs` app URL to `/workflows` (may need to communicate this to current users)
- Makes Workflows the logged-in home page
- Fix Workflow and Crawl UI access for viewer role (see follow-ups for caveats)

## Manual testing
1. Log in as user with crawler or higher access (e.g. org admin). Verify logged-in homepage is now Workflows
2. Regression testing: Verify you are able to run a crawl, edit a workflow, and create a new workflow as expected
3. Log out and log in as a user with only viewer access. Verify you are unable to see "Run Now", action menus, or "New Workflow" on the Workflows page
4. Click into a Workflow. Verify "Edit Workflow" is hidden
5. Click into a Crawl. Verify "Actions" menu and Metadata "edit pencil" button is hidden

### Screenshots
**Home view for users with "viewer" role:**
<img width="1127" alt="Screen Shot 2023-03-08 at 11 18 15 AM" src="https://user-images.githubusercontent.com/4672952/223821225-600b5731-b5c4-4f09-935a-68e8521caf67.png">

**Home view with users with "crawler" role:**
<img width="1128" alt="Screen Shot 2023-03-08 at 11 17 02 AM" src="https://user-images.githubusercontent.com/4672952/223821371-34c31582-8a97-4a4b-9edd-03ad567dc315.png">

### Follow-ups
This PR implements a few quick fixes to the crawls UI for users with the "viewer" role. It doesn't fix an existing issue where the exclusion editor is visible to users with a "viewer" role. This should be fixed in https://github.com/webrecorder/browsertrix-cloud/issues/603
